lib/builders/BrowserBundleBuilder.js

```diff
@@ -186,101 +210,105 @@ class BrowserBundleBuilder {
 					.bundle()
 				)
 			)
+			.then(() => new Promise((fulfill, reject) => this._createExternalsBundler()
+					.once('error', reject)
+					.once('bundle', bundleStream => bundleStream
+							.once('end', fulfill)
+							.on('error', reject)
+					)
+					.bundle()
+				)
+			)
+
 			.then(() => this._doPostBuildActions())
 			.then(() => this._isRelease ?
-					pfs.unlink(this._bootstrapperPath) :
+					pfs.unlink(this._appDefinitionsPath) :
 					this._watch()
 			)
 			.catch(reason => this._eventBus.emit('error', reason));
 	}
 
 	/**
-	 * Creates a bootstrapper file for the bundler.
+	 * Creates a app definitions file for the bundler.
 	 * @returns {Promise} The promise for finished work.
 	 * @private
 	 */
-	_createBootstrapper() {
+	_createAppDefinitions() {
 		return Promise.all([
 			this._storeFinder.find(),
 			this._componentFinder.find()
 		])
-			.then(found => this._bootstrapperBuilder.build(found[0], found[1]))
-			.then(realBootstrapper => {
-				if (realBootstrapper === this._bootstrapperCache) {
+			.then(found => this._appDefinitionsBuilder.build(found[0], found[1]))
+			.then(realAppDefinitions => {
+				if (realAppDefinitions === this._appDefinitionsCache) {
 					return null;
 				}
-				this._bootstrapperCache = realBootstrapper;
-				return pfs.writeFile(this._bootstrapperPath, realBootstrapper);
+				this._appDefinitionsCache = realAppDefinitions;
+				return pfs.writeFile(this._appDefinitionsPath, realAppDefinitions);
 			});
 	}
 
 	/**
-	 * Creates the browserify bundler or re-uses the existing one.
-	 * @returns {Browserify} The browserify instance.
+	 * Creates the browserify bundler for the app or re-uses the existing one.
+	 * @returns {Browserify} The browserify instances.
 	 * @private
 	 */
-	_createBundler() {
-		if (this._bundler) {
-			return this._bundler;
+	_createAppBundler() {
+		if (this._appBundler) {
+			return this._appBundler;
 		}
 
-		this._bundler = browserify([this._entryPath], {
+		this._appBundler = browserify(this._entryPath, {
 			cache: {},
 			packageCache: {},
-			debug: !this._isRelease
-		})
-			.transform(babelify, {
-				global: true,
-				ast: false,
-				comments: false,
-				sourceMap: !this._isRelease,
-				presets: [babelifyPreset]
-			});
+			debug: !this._isRelease,
+			filter: id => {
```

Comment:
I thought about it but browserify uses a dependency resolution library passing options to it and very unlikely they're going to change it.

But good point, maybe you should freeze on minor version.